config-qinstall.yml

```diff
@@ -0,0 +1,356 @@
+    idle_timeout: 300
+    # Set the max number of vm's in a VMSS; requires additional limit raise through support ticket for >100; 
+    # 100 is default value; lower numbers will improve scaling for single node jobs or jobs with small number of nodes
+    MaxScaleSetSize: 100
+  - name: hc44rs
+    vm_size: Standard_HC44rs
+    max_core_count: 440
+    image: azhpc:azhop-compute:centos-7_9:latest
+    spot: true
+    EnableAcceleratedNetworking: true
+  - name: hb120v2
+    vm_size: Standard_HB120rs_v2
+    max_core_count: 1200
+    image: azhpc:azhop-compute:centos-7_9:latest
+    spot: true
+    EnableAcceleratedNetworking: true
+  - name: hb120v3
+    vm_size: Standard_HB120rs_v3
+    max_core_count: 1200
+    image: azhpc:azhop-compute:centos-7_9:latest
+    spot: true
+    EnableAcceleratedNetworking: true
+    # Queue dedicated to GPU remote viz nodes. This name is fixed and can't be changed
+  - name: viz3d
+    vm_size: Standard_NV12s_v3
+    max_core_count: 48
+    # Use the pre-built azhop image from the marketplace
+    image: azhpc:azhop-desktop:centos-7_9:latest
+    # Use this image ID when building your own custom images
+    #image: /subscriptions/{{subscription_id}}/resourceGroups/{{resource_group}}/providers/Microsoft.Compute/galleries/{{sig_name}}/images/azhop-centos79-desktop3d/latest
+    ColocateNodes: false
+    spot: false
+    EnableAcceleratedNetworking: true
+    max_hours: 12 # Maximum session duration
+    min_hours: 1 # Minimum session duration - 0 is infinite
+    # Queue dedicated to share GPU remote viz nodes. This name is fixed and can't be changed
+  - name: largeviz3d
+    vm_size: Standard_NV48s_v3
+    max_core_count: 96
+    image: azhpc:azhop-desktop:centos-7_9:latest
+    ColocateNodes: false
+    EnableAcceleratedNetworking: true
+    spot: false
+    max_hours: 12
+    min_hours: 1
+    # Queue dedicated to non GPU remote viz nodes. This name is fixed and can't be changed
+  - name: viz
+    vm_size: Standard_D8s_v5
+    max_core_count: 200
+    image: azhpc:azhop-desktop:centos-7_9:latest
+    ColocateNodes: false
+    spot: false
+    EnableAcceleratedNetworking: true
+    max_hours: 12
+    min_hours: 1
+
+# Remote Visualization definitions
+enable_remote_winviz: false # Set to true to enable windows remote visualization
+
+remoteviz:
+  - name: winviz # This name is fixed and can't be changed
+    vm_size: Standard_NV12s_v3 # Standard_NV8as_v4 Only NVsv3 and NVsV4 are supported
+    max_core_count: 48
+    image: "MicrosoftWindowsDesktop:Windows-10:21h1-pron:latest"
+    ColocateNodes: false
+    spot: false
+    EnableAcceleratedNetworking: true
+
+# Application settings
+applications:
+  bc_codeserver:
+    enabled: true
+  bc_jupyter:
+    enabled: true
+  bc_amlsdk:
+    enabled: false
+  bc_rstudio:
+    enabled: false
+  bc_ansys_workbench:
+    enabled: false
+  bc_vmd:
+    enabled: false
+  bc_paraview:
```

Comment:
I would enabled paraview as it's used with the OpenFOAM tutorial

Reply:
Enabled